hedera-node/src/main/java/com/hedera/services/state/exports/SignedStateBalancesExporter.java

```diff
@@ -136,9 +136,10 @@ public void exportBalancesFrom(ServicesState signedState, Instant when) {
 		summary = summarized(signedState);
 		var expected = BigInteger.valueOf(expectedFloat);
 		if (!expected.equals(summary.getTotalFloat())) {
-			throw new IllegalStateException(String.format(
-					"Signed state @ %s had total balance %d not %d!",
-					when, summary.getTotalFloat(), expectedFloat)); }
+			throw new IllegalStateException(
+					"Signed state @ " + when + " had total balance "
+							+ summary.getTotalFloat() + " not " + expectedFloat + "!"
+			); }
```

Comment:
I don't think this is a hotspot either, I'm OK if we left it as it was, or we can use string concat. Either way is fine with me. The old code is nicer to read and more elegant, and frankly if we're hitting this exception frequently and it becomes a hotspot then we've got some serious problems :D.

Reply:
Haha yeah, reverted to old version.